pkg/sdn/registry/netnamespace/strategy.go

```diff
@@ -53,7 +53,7 @@ func (sdnStrategy) AllowUnconditionalUpdate() bool {
 
 // ValidateUpdate is the default update validation for a NetNamespace
 func (sdnStrategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) fielderrors.ValidationErrorList {
-	return validation.ValidateNetNamespaceUpdate(obj.(*api.NetNamespace), old.(*api.NetNamespace))
+	return fielderrors.ValidationErrorList{}
```

Comment:
at least call `ValidateObjectMetaUpdate` and delegate to `ValidateNetNamespace` to check validity of NetID, or you are allowing anything to be submitted (UID changes, completely invalid NetID, etc)